src/views/AnnualReport.vue

```diff
@@ -516,36 +516,40 @@ export default {
     }
   },
 
-  async mounted (): Promise<void> {
-    if (this.filingId > 0) {
-      this.loadingMessage = `Resuming Your ${this.ARFilingYear} Annual Report`
-      // resume draft filing
-      await this.fetchDraftFiling()
-      // fetch original office addresses and directors
-      // update working data only if it wasn't in the draft
-      if (!this.isJestRunning) {
-        await this.$refs.officeAddressesComponent.getOrigAddresses(this.asOfDate, isEmpty(this.updatedAddresses))
-        await this.$refs.directorsComponent.getOrigDirectors(this.asOfDate, isEmpty(this.updatedDirectors))
-      }
-    } else {
-      this.loadingMessage = `Preparing Your ${this.ARFilingYear} Annual Report`
-      // this is a new filing
-      // fetch original office addresses and directors + update working data
-      if (!this.isJestRunning) {
-        await this.$refs.officeAddressesComponent.getOrigAddresses(this.asOfDate, true)
-        await this.$refs.directorsComponent.getOrigDirectors(this.asOfDate, true)
+  mounted (): void {
+    // wait until entire view is rendered (including all child components)
+    // see https://v3.vuejs.org/api/options-lifecycle-hooks.html#mounted
+    this.$nextTick(async () => {
```

Comment:
If you "hide whitespace changes" in the gear menu above, you will see that I only wrapped the previous code in "nextTick".

Reply:
Thank you for this tip. Farewell forever whitespace changes..